DataFormats/L1TCalorimeterPhase2/src/DigitizedClusterCorrelator.cc

```diff
@@ -0,0 +1,9 @@
+#include <bitset>
+
+#include "DataFormats/L1TCalorimeterPhase2/interface/DigitizedClusterCorrelator.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+void l1tp2::DigitizedClusterCorrelator::print(const std::string location) {
+  //edm::LogInfo("DigitizedClusterCorrelator") << "Info: DigitizedClusterCorrelator " << location << ": " << (int) data() << std::endl;
+  std::cout << "[Info:] DigitizedClusterCorrelator " << location << ": " << std::bitset<64>{data()} << std::endl;
+}
```

Comment:
This isn't allowable in CMSSW. The LogInfo is better, but please do not leave commented code.

Reply:
Removed method in [e7e753d](https://github.com/cms-l1t-offline/cmssw/pull/1069/commits/e7e753df69e7aff845a3b3dfbaed9a06c9fdaec6)